projects/demo/src/modules/components/prompt/examples/1/index.ts

```diff
@@ -0,0 +1,37 @@
+import {Component, Inject} from '@angular/core';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+import {TuiAlertService, TuiDialogService} from '@taiga-ui/core';
+import {TUI_PROMPT, TuiPromptData} from '@taiga-ui/kit';
+import {switchMap} from 'rxjs/operators';
+
+@Component({
+    selector: 'tui-prompt-example-1',
+    templateUrl: './index.html',
+    changeDetection,
+    encapsulation,
+})
+export class TuiPromptExample1 {
+    constructor(
+        @Inject(TuiDialogService) private readonly dialogs: TuiDialogService,
+        @Inject(TuiAlertService) private readonly alerts: TuiAlertService,
+    ) {}
+
+    onClick(): void {
+        const data: TuiPromptData = {
+            content:
+                'This is <code>PolymorpheusContent</code> so it can be template too!',
+            yes: 'That is great!',
+            no: 'Who cares?',
```

Comment:
I am not sure that `prompt` is appropriate naming.

See description of native [window.prompt](https://developer.mozilla.org/ru/docs/Web/API/Window/prompt):
> window.prompt() instructs the browser to display a dialog with an optional message prompting the user **to input some text**, and to wait until the user either submits the text or cancels the dialog.

Do you mean `TUI_CONFIRM` ?
See description of native [window. confirm](https://developer.mozilla.org/en-US/docs/Web/API/Window/confirm):

> window.confirm() instructs the browser to display a dialog with an optional message, and to wait until the user either **confirms or cancels** the dialog.

Reply:
Oops, this component already exists ...

Reply:
Should we rename it in future major release ?

Reply:
Sure, added a TODO